sql/hive/src/test/scala/org/apache/spark/sql/hive/test/TestHive.scala

```diff
@@ -534,7 +534,8 @@ private[hive] class TestHiveSparkSession(
       }
 
       // Clean out the Hive warehouse between each suite
-      val warehouseDir = new File(new URI(sparkContext.conf.get("spark.sql.warehouse.dir")).getPath)
+      val warehouseDir = new File(new URI(
+        sparkContext.conf.get(WAREHOUSE_PATH.key)).getPath)
```

Comment:
IIRC, I gave one-liner. (https://github.com/apache/spark/pull/25059#discussion_r302307831).
Why do you keep two lines?